applications/ila-cookie-banner/ila-cookie-banner.css

```diff
@@ -0,0 +1,128 @@
+:root {
+    --ila-cookieb-margin: .25rem;
+    --ila-cookieb-padding-x: 1rem;
+    --ila-cookieb-padding-y: .3rem;
+    --ila-cookieb-background-blue: #13294b;
+    --ila-cookieb-font-white: #ffffff;
+    --ila-cookieb-hover-orange: #ff5f05;
+}
+
+.cookieb-holder {
+    position: fixed;
+    bottom: 0;
+    left: 0rem;
+    width: auto;
+}
+
+.ila-cookieb__cookieb {
+    border: 1px solid var(--il-gray-2);
+    margin: var(--ila-cookieb-margin);
+    padding: var(--ila-cookieb-padding-y) var(--ila-cookieb-padding-x);
+    background-color: white;
+    position: relative;
+}
+
+.ila-cookieb__h3 {
+    margin: 0 0 auto auto;
+}
+
+.ila-cookieb__content p {
+    margin: 0;
+    padding: 0;
+    line-height: 1.5rem;
+    width: 60%;
+}
+
+.ila-cookieb__content {
+    margin: 10px 0;
+    display: flex;
+    flex-wrap: wrap;
+}
+
+.ila-cookieb__actions {
+    padding-top: 1rem;
+    padding-left: 1rem;
+}
+
+.ila-cookieb__button {
+    margin: 0 0 .5rem .5rem;
+}
+
+.ila-cookieb__close-button {
+    position: absolute;
+    right: 0;
+    top: 0;
+    margin-top: calc(var(--ila-cookieb-padding-y) - 5px);
+    margin-right: calc(var(--ila-cookieb-padding-x) - 5px);
+    padding: 0;
+    background: transparent;
+    border: none;
+    cursor: pointer;
+    color: var(--ila-cookieb-font-white);
+}
+
+.ila-cookieb__close-button:hover,
+.ila-cookieb__close-button:focus {
+    transition: background-color 0.3s;
+    background-color: var(--ila-cookieb-hover-orange);
+    color: white;
+}
+
+.ila-cookieb__button:hover,
+.ila-cookieb__button:focus {
+    background-color: var(--ila-cookieb-hover-orange);
+}
+
+
+.ila-cookieb__close-icon {
+    padding: var(--ila-cookieb-padding-y) var(--ila-cookieb-padding-x);
+    font-size: 25px;
+}
+
+.ila-cookieb.ila-cookieb--closed {
+    transition: opacity 0.25s ease-in-out, visibility 0.25s, order 1s;
```

Comment:
Personal preference, but would add a `@media prefers-reduced-motion` query here

Reply:
Thanks! That's a new one, but we've added this, to remove the transition when `prefers-reduced-motion` is applied.